vscode/microsoft-kiota/package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "Microsoft Kiota",
   "publisher": "ms-graph",
   "description": "Client generator for HTTP REST APIs described by OpenAPI which helps eliminate the need to take a dependency on a different API client for every API that you need to call, as well as limiting the generation to the exact API surface area you're interested in, thanks to a filtering capability.",
-  "version": "1.1.3",
+  "version": "1.2.0",
```

Comment:
Should this also be 1.2.1?

Reply:
this only need to be updated for local debugging purposes, the CD updates the version for us inline with the csproj.